<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunningInSetting</class>
 <widget class="QDialog" name="RunningInSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(27, 150, 208);
}
QLabel
{
font-size:26px;
color: rgb(255, 255, 255);
}
QPushButton{
border-radius:12px;
background-color: rgb(80, 80, 80);
font-size:22px;
color:rgb(255, 255, 0);
}
QPushButton::checked{
color:red;
}
QPushButton::pressed{
background-color: rgb(120, 120, 120);
}

QToolButton{
font-size:26px;
font:bold;
color:blue;
background-color:transparent;
border:noborder;
}
QToolButton::checked{
color:red;
}</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;
font-size:28px;</string>
   </property>
   <property name="text">
    <string>设备磨合设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>磨合时长(天)</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tlb_runindays">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>热机温度1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>热机限速1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tlb_tempre">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>1</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tlb_speedlimit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>保
存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>180</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
</string>
   </property>
   <property name="text">
    <string>开
始
磨
合</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tips">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_remainingtime">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="tlb_runinspeed">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>磨合限速</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tlb_heatclothnum">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>热机件数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>热机状态</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tlb_heatflag">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tlb_speedlimit_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tlb_tempre_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>1</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>热机温度2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>热机限速2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
